tests/python-pytest/onnx/test_onnxruntime_cv.py

```diff
@@ -315,6 +317,11 @@ def normalize_image(imgfile):
                     assert_obj_detetion_result(mx_class_ids[0], mx_scores[0], mx_boxes[0],
                                                onnx_class_ids[0], onnx_scores[0], onnx_boxes[0],
                                                score_thresh=0.8, score_tol=0.05, box_tol=15)
+                elif model.startswith('faster_rcnn_fpn'):
+                    onnx_class_ids, onnx_scores, onnx_boxes = session.run([], {input_name: img_data.asnumpy()})
+                    assert_obj_detetion_result(mx_class_ids[0], mx_scores[0], mx_boxes[0],
+                                               onnx_class_ids[0], onnx_scores[0], onnx_boxes[0],
+                                               score_thresh=0.8, score_tol=0.05, box_tol=30)
```

Comment:
Other object detection models are able to pass the test with `box_tol=0.01`, but here we loose it to `box_tol=30` for faster_rcnn models. Do we know what causes the accuracy regression here?

Reply:
I think the onnx model can predict the same objects with very similar scores. It's just with a few boxes one of the four values (xstart, xend, ystart, yend) can differ by quit a bit (refer to the bbox of the boy in red). I am not sure about the cause, but it must be related to the fpn structure.


![soccer_onnx](https://user-images.githubusercontent.com/16669457/115301423-06c72900-a116-11eb-86c5-2ed362d0a3cd.png))

![soccer_mx](https://user-images.githubusercontent.com/16669457/115301411-029b0b80-a116-11eb-84ee-aa8a81dd54f0.png)